generators/client/templates/angular/package.json.ejs

```diff
@@ -165,7 +166,7 @@
     <%_ } _%>
     "jest": "jest --coverage --logHeapUsage --maxWorkers=2 --config jest.conf.js",
     "start": "ng serve<% if (!microfrontend) { %> --hmr<% } %>",
-    "start-tls": "<%= clientPackageManager %> run webapp:dev -- --env.tls",
+    "start-tls": "cross-env TLS=true <%= clientPackageManager %> run webapp:dev-ssl",
```

Comment:
Why TLS env variable?

Reply:
Based on the TLS variable proxy and browser-sync have been configured. as it is passed to ng serve. ng serve directly not accepting --env.tls as it is not a valid option. that is why I changed it. Please find below

https://github.com/jhipster/generator-jhipster/blob/main/generators/client/templates/angular/webpack/proxy.conf.js.ejs#L20
https://github.com/jhipster/generator-jhipster/blob/main/generators/client/templates/angular/webpack/proxy.conf.js.ejs#L38

https://github.com/jhipster/generator-jhipster/blob/main/generators/client/templates/angular/webpack/webpack.custom.js.ejs#L34
https://github.com/jhipster/generator-jhipster/blob/main/generators/client/templates/angular/webpack/webpack.custom.js.ejs#L82

Reply:
You can use targetOptions or config to detect ssl option and drop TLS variable.
https://github.com/jhipster/generator-jhipster/blob/8f6769669e102b83d3f7a5fe959981b2d5e8df83/generators/client/templates/angular/webpack/webpack.custom.js.ejs#L74

Reply:
Updated Implementation with 'config'. Please check now